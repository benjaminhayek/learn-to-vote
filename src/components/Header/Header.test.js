import React from 'react';
import { Header } from './index';
import { shallow } from 'enzyme';

describe('HEADER', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<Header />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})