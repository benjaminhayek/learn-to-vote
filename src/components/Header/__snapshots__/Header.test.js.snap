// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADER should match the snapshot 1`] = `
<div
  className="header"
>
  <h1
    className="header-title"
  >
    Educate Yourself on Education!
  </h1>
  <Link
    replace={false}
    to="/senators"
  >
    <button
      className="switch"
    >
      <img
        alt="debate-pic"
        className="debate-pic"
        src="debate2.svg"
      />
      <h2
        className="instructs"
      >
        Click here to see senators
      </h2>
    </button>
  </Link>
</div>
`;
